src/components/Nav/Nav.js

```diff
@@ -46,39 +61,44 @@ class Nav extends Component {
   };
 
   render() {
-    const { isSignup, isLogin, userIsLoggedIn } = this.state;
+    const { isSignup, isLogin, searchData } = this.state;
 
     var loginComponent = (
       <>
         <button
           className='loginBtn'
-          onClick={(e) => this.handleLoginOrSignupModal(e)}
-        >
+          onClick={(e) => this.handleLoginOrSignupModal(e)}>
           로그인
         </button>
         <button
           className='signupBtn'
-          onClick={(e) => this.handleLoginOrSignupModal(e)}
-        >
+          onClick={(e) => this.handleLoginOrSignupModal(e)}>
```

Comment:
```suggestion
          onClick={this.handleLoginOrSignupModal}>
```
이벤트 객체가 그대로 전달될 경우 생략할 수 있습니다!

Reply:
바로 수정하였습니다 감사합니다 :)